src/torchcodec/_core/Transform.cpp

```diff
@@ -57,4 +57,25 @@ int ResizeTransform::getSwsFlags() const {
   return toSwsInterpolation(interpolationMode_);
 }
 
+CropTransform::CropTransform(const FrameDims& dims, int x, int y)
+    : outputDims_(dims), x_(x), y_(y) {
+  TORCH_CHECK(x_ >= 0, "Crop x position must be >= 0, got: ", x_);
+  TORCH_CHECK(y_ >= 0, "Crop y position must be >= 0, got: ", y_);
+}
+
+std::string CropTransform::getFilterGraphCpu() const {
+  return "crop=" + std::to_string(outputDims_.width) + ":" +
+      std::to_string(outputDims_.height) + ":" + std::to_string(x_) + ":" +
+      std::to_string(y_) + ":exact=1";
```

Comment:
On design discussion point 1: agreed this should be hardcoded to `exact=1`, and *not* a parameter. And I think this is the case regardless of what torchvision does. With `exact=0` we risk the output frames not being of the desired dimensions, and that potentially means a downstream model failing loudly because its expected input dimensions weren't respected.

I would suggest to go slightly further and not even include "exact=1" in the file names (like `nasa_13013.mp4.crop=300:200:50:35:exact=1.stream3.frame000000.pt`), because I don't think we'll be enabling exact=0.